ani-cli

```diff
@@ -447,12 +477,14 @@ player_fn="mpv" #video player needs to be able to play urls
 is_download=0
 PID=0
 quality=best
+idx=5
 scrape=query
 download_dir="."
 choice=
 auto_play=0
 # history file path
 logfile="${XDG_CACHE_HOME:-$HOME/.cache}/ani-hsts"
+urlfile="${XDG_CACHE_HOME:-$HOME/.cache}/ani-url"
```

Comment:
Has someone checked if this even speeds up anything?
I could imagine a storage read being not that much faster than a network call.
At least not in a way where it would justify several loc

Reply:
well.. read from local storage is always faster than fetching from internet.. also we are just storing latest proxy used by gogoplay to make sure we won't get shut out on dead url..

Reply:
Then it won't hurt